mmdet/ops/dcn/modules/deform_conv.py

```diff
@@ -0,0 +1,45 @@
+import math
+
+import torch
+import torch.nn as nn
+from torch.nn.modules.module import Module
+from torch.nn.modules.utils import _pair
+
+from ..functions.deform_conv import deform_conv
+
+
+class DeformConv(Module):
```

Comment:
To be consistent with other modules, use `DeformConv(nn.Module)`, and `from torch.nn.modules.module import Module` is unnecessary.